src/FocusTrap.ts

```diff
@@ -56,7 +60,10 @@ export const FocusTrap = defineComponent({
             // has no effect if already activated
             trap = createFocusTrap(el.value, {
               escapeDeactivates: props.escapeDeactivates,
-              allowOutsideClick: () => props.allowOutsideClick,
+              allowOutsideClick: (e: MouseEvent | TouchEvent) =>
```

Comment:
Are the explicit types necessary here? Usually these get inferred automatically

Reply:
My IDE was yelling at me, but let me give this another shot

Reply:
Looks like it works without the explicit typing. I wonder if it was able to figure it out _after_ I swapped to using the provided `MouseEventToBoolean` type.

I've updated the PR to remove the explicit typing 👍